index.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
-module.exports = function toFastProperties(obj) {
-	function f() {}
-	f.prototype = obj;
-	new f();
-	return;
-	eval(obj);
-};
+module.exports = function toFastproperties(o) {
+	function Sub() {}
+	Sub.prototype = o;
+	var receiver = new Sub(); // create an instance
+	function ic() { return typeof receiver.foo; } // perform access
```

Comment:
Here is the actual change:

https://github.com/v8/v8/commit/be0494ba5bfdc8f7b225372ed7990a06dca7ad46 

IC typically means instruction cache - so I assume that's it.